### PR TITLE
py3: Base: files A-I ported to python3

### PR DESCRIPTION
**Pull request by looooo**
issue 0000995

### CODE REVIEW

File: src/Base/CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +30,16 @@ if(MSVC)
         Rpcrt4.lib
         version.lib
     )
+
+    #Universal C runtime introduced in VS 2015 (cl version 19)
```
Comment (wwmayer) [inline src/Base/CMakeLists.txt]: This check fails. Here I have VS 2013 and the output of the variable is "18.0.40629.0" which is already greater than "18"
It looks lie cmake has no "VERSION EQUAL or GREATER" so you have to use "NOT VERSION_LESS" than "19"

Comment (looooo) [inline src/Base/CMakeLists.txt]: "NOT VERSION_LESS" seems to not work either.

Comment (wwmayer) [inline src/Base/CMakeLists.txt]: It's the wrong syntax. It should be 
`if (NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS "19")`

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**